app/services/order_row_importer.rb

```diff
@@ -295,7 +296,17 @@ def validate_existing_order
     end
   end
 
-  # Projects adds a custom attribute and overrides this method to validate it
   def validate_custom_attributes
+    if field(:project_name).present? && project.nil?
+      add_error(:project_not_found)
+    end
+  end
+
+  def project
+    facility.projects.active.find_by name: field(:project_name)
+  end
+
+  def custom_attributes
+    { project_id: project&.id }
```

Comment:
these aren't really custom attributes any more, right?  and we already define `custom_attributes` on line 162.  How about adding `project_id` to `to_add_attributes` instead?

Also, I think `custom_attributes` and `validate_custom_attributes` are probably un-used now, so we should add code comments to the methods